spec/models/item_spec.rb

```diff
@@ -14,6 +14,16 @@
     it { should have_many(:orders).through(:order_items) }
   end
 
+
+  it 'model methods' do
```

Comment:
This should move down to the instance methods section since you are calling .disable on an instance of an Item.  Use the .average_fulfilled_time as a template for the describe / it blocks.

Reply:
done!